eng/Version.Details.xml

```diff
@@ -1,12 +1,12 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Diagnostics.Monitoring" Version="6.0.0-preview.23056.1">
```

Comment:
This was the main source of conflicts. We wouldn't be using this version anyway, but I can change this to reflect the latest shipped bits.